test/address.spec.ts

```diff
@@ -76,4 +76,12 @@ describe('Address', () => {
       assert.equal(addr.toString(), result)
     }
   })
+
+  it('should provide a buffer that does not mutate the original address', () => {
+    const str = '0x2f015c60e0be116b1f0cd534704db9c92118fb6a'
+    const address = Address.fromString(str)
+    let addressBuf = address.toBuffer()
+    addressBuf = Buffer.from('test')
```

Comment:
This doesn't work: `Buffer.from` copies the Buffer. What we want is to actually call a method on this Buffer so it overwrites is (this basically sets `addressBuf` to some other value). For instance we can use `addressBuf.fill(0xff)`.

Example:

```
> bfr = Buffer.from('aaaaaa', 'hex')
<Buffer aa aa aa>
> bfr.fill(0)
<Buffer 00 00 00>
> bfr
<Buffer 00 00 00>
```

Reply:
oh nice thanks for explaining, that makes sense. fixed in https://github.com/ethereumjs/ethereumjs-util/pull/277/commits/18b9deadaa4399d4125c42538e7254910117cb45